docs/content/preview/yugabyte-voyager/install-yb-voyager.md

```diff
@@ -106,7 +106,43 @@ Install yb-voyager on a machine which satisfies the [Prerequisites](#prerequisit
   </div>
 </div>
 
-### Collect diagnostics
+## Upgrade yb-voyager
+
+{{< tabpane text=true >}}
```

Comment:
For the tabs, use the same style as the tabs in Install section above

Reply:
How do you upgrade the other OSs that in Install section

Reply:
Hey @aishwarya24 we are not providing any upgrade commands for the other installation methods yet. So is it possible to limit the new section only to RHEL and Ubuntu?
Maybe we can remove the tabs to select between the two and just add the content in the respective sections?

Reply:
@ShivanshGahlot - sure ill include it in the install section at the moment.. Making the change.